filters/scanner_test.go

```diff
@@ -324,14 +330,12 @@ func TestScanner(t *testing.T) {
 		t.Run(testcase.name, func(t *testing.T) {
 			var sc scanner
 			sc.init(testcase.input)
-
```

Comment:
Let's reduce changes you don't need. I don't have strong opinions on whitespace changes themselves, but they shouldn't be in this PR.

Reply:
I have revoked the whitespace changes as you said.